extensions/blocks/calendly/calendly.php

```diff
@@ -160,3 +152,85 @@ function get_attribute( $attributes, $attribute_name ) {
 		return $default_attributes[ $attribute_name ];
 	}
 }
+
+/**
+ * Enqueues the Calendly JS library and inline function to attach event
+ * handlers to the button.
+ *
+ * @return void
+ */
+function enqueue_calendly_js() {
+	wp_enqueue_script(
+		'jetpack-calendly-external-js',
+		'https://assets.calendly.com/assets/external/widget.js',
+		null,
+		JETPACK_VERSION,
```

Comment:
Quick change, it should be `JETPACK__VERSION` here.

```suggestion
		JETPACK__VERSION,
```